Orange/canvas/application/outputview.py

```diff
@@ -22,14 +22,8 @@ def __init__(self, *args, **kwargs):
         self.setSizePolicy(QSizePolicy.Preferred, QSizePolicy.Preferred)
 
         font = self.font()
-        if hasattr(QFont, "Monospace"):
-            # Why is this not available on Debian squeeze
-            font.setStyleHint(QFont.Monospace)
-        else:
-            font.setStyleHint(QFont.Courier)
-
-        font.setFamily("Monaco")
-        font.setPointSize(12)
+        font.setStyleHint(QFont.Monospace)
```

Comment:
Squeeze is no longer supported. Obviously Squeeze at that time had too old Qt that didn't yet contain `QFont.Monospace`.